src/store/modules/gameList.js

```diff
@@ -1,12 +1,13 @@
 import { cloneDeep } from 'lodash';
 import { io } from '@/plugins/sails.js';
+import GameStatus  from '../../../utils/GameStatus.json';
 
 class GameSummary {
   constructor(obj) {
     this.id = obj.id ? obj.id : null;
     this.name = obj.name ? obj.name : null;
     this.numPlayers = Object.prototype.hasOwnProperty.call(obj, 'players') ? obj.players.length : 0;
-    this.status = Object.prototype.hasOwnProperty.call(obj, 'status') ? obj.status : false;
+    this.status = Object.prototype.hasOwnProperty.call(obj, 'status') ? obj.status : GameStatus.STARTED;
```

Comment:
I like that this makes games default to unavailable to join as a player but it means they could be spectate or, even though the status being missing means something is wrong. I think it makes sense for them to default to archived instead so that if this data is missing the game doesn't appear anywhere 
```suggestion
    this.status = Object.prototype.hasOwnProperty.call(obj, 'status') ? obj.status : GameStatus.ARCHIVED;
```

Reply:
I would prefer archived also, if the app isn't sure just don't even give the option to let the user do anything with it.